packages/forma-36-website/gatsby-config.js

```diff
@@ -361,6 +331,36 @@ module.exports = {
             name: 'Tooltip',
             link: '/components/tooltip/',
           },
+          {
+            name: 'Typography Components',
+            link: '',
+            menuLinks: [
+              {
+                name: 'Text',
+                link: '/components/text/',
+              },
+              {
+                name: 'Paragraph',
+                link: '/components/paragraph/',
+              },
+              {
+                name: 'Heading',
+                link: '/components/heading/',
+              },
+              {
+                name: 'SectionHeading',
+                link: '/components/section-heading/',
+              },
+              {
+                name: 'Subheading',
+                link: '/components/subheading/',
+              },
+              {
+                name: 'DisplayText',
+                link: '/components/display-text/',
+              },
+            ],
+          },
```

Comment:
I think there are more stuff to be sorted here, look:

![Screenshot 2021-11-12 at 15 38 08](https://user-images.githubusercontent.com/6597467/141484314-25bf37ea-4205-4f80-a26b-3611d68c1e75.png)

for example: Popover is after DateTime, but before EntityList

Reply:
ah, true! sorry, didn't notice. on my way.

Reply:
hmm, now when I think about it, maybe Popover should be a subnav of Menu?

Reply:
I only gave one example, there are a bunch of components that are not in the right alphabetical order

IMO the way Popover and Menu are now is fine

Reply:
I would not make Popover as a subnav of Menu

Reply:
I guess should be ok now?
<img width="276" alt="Screenshot 2021-11-12 at 16 11 44" src="https://user-images.githubusercontent.com/4272331/141489270-8bbcfdcb-3813-462f-8fe8-9ff17fa00ee5.png">